src/firstvoices/backend/models/part_of_speech.py

```diff
@@ -29,6 +33,12 @@ class PartOfSpeech(BaseModel):
     class Meta:
         verbose_name = _("Part Of Speech")
         verbose_name_plural = _("Parts Of Speech")
+        rules_permissions = {
+            "view": rules.always_allow,
+            "add": predicates.is_superadmin,
+            "change": predicates.is_superadmin,
+            "delete": predicates.is_superadmin,
+        }
```

Comment:
+1 for these permission settings. Looks good.

Reply:
Is there any situation where a staff user might need to add to or edit parts of speech?

Reply:
Is there any situation where a staff user might need to add to or edit parts of speech?